Dockerfile

```diff
@@ -4,39 +4,39 @@ WORKDIR /workspace
 COPY schematics /workspace/schematics/
 COPY package.json package-lock.json /workspace/
 RUN npm i --ignore-scripts
+RUN npm run ngcc
 COPY projects/organization-management/src/app /workspace/projects/organization-management/src/app/
 COPY projects/requisition-management/src/app /workspace/projects/requisition-management/src/app/
 COPY src /workspace/src/
 COPY tsconfig.app.json tsconfig.app-no-checks.json tsconfig.json ngsw-config.json .browserslistrc angular.json /workspace/
 RUN npm run build:schematics && npm run synchronize-lazy-components -- --ci
-ARG configuration=production
 COPY scripts /workspace/scripts/
-RUN test "${configuration}" = 'local' && node scripts/init-local-environment.js || true
+RUN npm run init-development-environment -- --empty
 ARG serviceWorker
 RUN node schematics/customization/service-worker ${serviceWorker} || true
 COPY templates/webpack/* /workspace/templates/webpack/
 ARG testing=false
 ENV TESTING=${testing}
-RUN npm run ng -- build -c ${configuration} --deploy-url=DEPLOY_URL_PLACEHOLDER
 # synchronize-marker:pwa-docker-build:end
 
 # ^ this part above is copied to Dockerfile_noSSR and should be kept in sync
 
+RUN npm run build:multi client -- --deploy-url=DEPLOY_URL_PLACEHOLDER
 COPY tsconfig.server.json server.ts /workspace/
-RUN npm run ng -- run intershop-pwa:server -c ${configuration}
+RUN npm run build:multi server
 # remove cache check for resources (especially index.html)
 # https://github.com/angular/angular/issues/23613#issuecomment-415886919
 RUN test "${serviceWorker}" = "true" && sed -i 's/canonicalHash !== cacheBustedHash/false/g' /workspace/dist/browser/ngsw-worker.js || true
-COPY dist/entrypoint.sh dist/healthcheck.js dist/server.crt dist/server.key dist/robots.txt* /workspace/dist/
+COPY dist/* /workspace/dist/
 
 FROM node:14-alpine
+RUN npm i -g express express-http-proxy pm2 prom-client
 COPY --from=buildstep /workspace/dist /dist
 ARG displayVersion=
 LABEL displayVersion="${displayVersion}"
-ENV DISPLAY_VERSION=${displayVersion}
-ARG configuration=production
-LABEL configuration="${configuration}"
+ENV DISPLAY_VERSION=${displayVersion} NODE_PATH=/usr/local/lib/node_modules
 EXPOSE 4200
+RUN mkdir /.pm2 && chmod 777 -Rf /.pm2 && touch /dist/ecosystem.yml && chmod 777 -f /dist/ecosystem.yml
```

Comment:
Why do we need rwx on `dist/ecosystem.yml`? Is pm2 writing into that file during runtime?

Reply:
Just read is enough.

Reply:
Oh wait, the entrypoint is writing this as nobody, so yes, rw is required